website/docs/r/container_service.html.markdown

```diff
@@ -179,6 +179,8 @@ The following arguments are supported:
 
 * `count` - (Required) Number of masters (VMs) in the container service cluster. Allowed values are 1, 3, and 5. The default value is 1.
 * `dns_prefix` - (Required) The DNS Prefix to use for the Container Service master nodes.
+* `vm_size` - (Required) The VM Size of each of the masters (VMs) (e.g. Standard_F1 / Standard_D2v2).
```

Comment:
I believe this is marked optional in schema, not sure which is correct.

Reply:
Thanks! It is `optional`, fixed documentation.